src/lib/category.mli

```diff
@@ -0,0 +1,56 @@
+
+(* This file is free software, part of FTW. See file "LICENSE" for more information *)
+
+(* Type definitions *)
+(* ************************************************************************* *)
+
+type non_competitive =
+  | Regular
+  | Qualifying
+  | Invited (**)
+[@@deriving yojson]
+(** Non competitive competitions, i.e. does not require or give points, but
+    may grant access to some competititve divisions in some cases related
+    to the creation of the point system. *)
+
+type t =
+  | Competitive of Division.t
+  | Non_competitive of non_competitive (**)
+[@@deriving yojson]
```

Comment:
`t` and `non_competitive` duplicated with `src/li/category.ml`

Reply:
It's an unfortunate limitation of OCaml's separate compilation that types need to be duplicated in the implementation (in `.ml` file) and the interface (the `.mli` file).